plugin-utilities-plugin/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+group = rootProject.group
+version = rootProject.version
+
+dependencies {
+    implementation("com.xenomachina:kotlin-argparser:2.0.7")
+}
+
+// TODO: move into the project
+open class CliTask : org.jetbrains.intellij.tasks.RunIdeTask() {
```

Comment:
I don't think that this code can be easily reused when creating CLI runners that accept extra arguments. In my case, `PreprocessingRunner` should accept path to Android SDK and I do not see how I can effectively reuse this code. If I extend this class I will still have the `runner` field which I do not need. Also, my runner will require a new entrypoint like `./gradlew -q preprocess`

Reply:
We discussed it in DM. Should we resolve it or not?